pingplot.py

```diff
@@ -18,8 +18,13 @@
 # ping
 def pinger(host, n):
     """Executes the PCs ping command"""
-    proc = os.popen(f"ping -{ping_flag} {n} {host}")
-    return ''.join(proc.readlines())
+    if sys.platform != 'win32':
+        proc = os.popen(f"/sbin/ping -{ping_flag} {n} {host}")
```

Comment:
This likely won't work for Linux, as `ping` may be in `/bin` as well.

I haven't tried this, but does the program not work for MacOS?

Reply:
It's OK to just:
```
proc = os.popen(f"ping -{ping_flag} {n} {host}")
```
The reason why i used /sbin/ping instead was to work with cron: https://stackoverflow.com/questions/53345985/issue-pinging-with-python-script-running-as-cron-job